### PR TITLE
Upgrade SDK to v0.41.4-akash-2

### PR DESCRIPTION
**Pull request by akhilkumarpilli**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,5 +14,6 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-# vendor/
+vendor/
```
Comment (troian) [inline .gitignore:17]: do we need to commit vendor?

Comment (akhilkumarpilli) [reply .gitignore:17]: Ohh, sorry. I thought this will ignore vendor folder.

Comment (boz) [reply .gitignore:17]: We should not commit vendor - :+1: to this change.

File: .gitignore [hunk 2]
```diff
@@ -14,6 +14,6 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-vendor/
+# vendor/
```
Comment (boz) [inline .gitignore]: It should be ignored like you had it in the original PR

File: .gitignore (hunk 1 above)
Comment (troian) [reply .gitignore:17]: sorry, i somehow though this is an akash repo.
so this change is good

**Review by anilcse — APPROVED** [review]
lgtm

**Pull request merged**